lib/components/table.vue

```diff
@@ -318,6 +320,29 @@
         opacity: 0.3;
     }
 
+    table.table-sm thead .sorting:before,
```

Comment:
Also not sure why pagination is being imported, as it is not used directly by table.